inscriptions/ISic003508.xml

```diff
@@ -5,7 +5,7 @@
     <teiHeader>
         <fileDesc>
             <titleStmt>
-                <title>I.Sicily inscription 003508</title>
+                <title>Epitaph for Velleia Vesonia</title>
```

Comment:
this record appears to have got confused! ISic3508 (TM284778) is I.MusMessina I (i.e. Roman i, not Arabic 1), cast of half a tablet, for which the text transmitted by PHI is correct, but the one here included from EDR is erroneous (IMessina 1 instead), and should be EDR033620 (yes, it was a stupid idea of Bitto's to create a corpus that used both arabic and roman numerals for different inscriptiosn, and it has caused chaos...). The text of Velleia Vesonia belongs in http://sicily.classics.ox.ac.uk/inscription/ISic003433